### PR TITLE
Add advanced alignment to inspector

### PR DESCRIPTION
**Pull request by ellatrix**
Alternatively I can remove the centre alignment from the inline toolbars as well. I'm fine with either.

### CODE REVIEW

**Comment by jasmussen** [comment]
Thanks for exploring, it has helped clarify that we need some more clear patterns for what goes in the inspector and what goes on the block, and how to make room for more buttons especially with mobile as a consideration. 

I need to work on some inspector work for the gallery in the next couple of days anyway, so perhaps it would be good to press pause on these explorations for now, and resume once we have some patterns hashed out. 

Right now I'm thinking we need more clear examples of what constitutes "basic", and what constitutes "advanced". We may need some new quick toolbar patterns to make sure things fit on mobile, perhaps ressurect the idea of collapsing some groups (like alignments, block styles, or heading levels) into dropdowns when there's not enough room, as shown here: https://github.com/WordPress/gutenberg/issues/860#issuecomment-305737751

File: blocks/library/text/index.js [hunk 1]
```diff
@@ -3,6 +3,7 @@
  */
 import { __ } from 'i18n';
 import { concatChildren } from 'element';
+import { Toolbar } from 'components';
```
Comment (aduth) [inline blocks/library/text/index.js]: Note with #2172, these need to be updated to prefix the dependencies with `@wordpress/`. You will need to perform a rebase against the latest version of master and apply your changes:

```
git fetch origin
git rebase origin/master
```

**Comment by gziolo** [comment]
@mtias or @karmatosed is it still relevant? It seems like an useful addition.

**Comment by iseulde** [comment]
This PR just explores keeping only one alignment button inline that toggles centre and default alignment, since these are the most used, and moves explicit left, and right alignment to the inspector.

**Comment by jasmussen** [comment]
I think recent explorations that surround the toolbar, as well as toolbar overflow, are probably higher on our list for the time being. If those work, we can close this. If they don't work, then we should totally revisit this.

**Comment by gziolo** [comment]
I will close it then, we can always reopen later if we change our mind :)

**Pull request closed without merging**